http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -531,6 +533,15 @@ else if ((cause instanceof NoClassDefFoundError) && cause.getMessage().endsWith(
             value = responseHandler.handle(request, jettyResponse);
         }
         finally {
+            if (jettyResponse != null && jettyResponse.getStatusCode() == 200) {
+                try (InputStream inputStream = jettyResponse.getInputStream()) {
+                    ByteStreams.exhaust(inputStream);
+                }
+                catch (IOException ignored) {
+                    // if the response processing didn't throw, don't throw
```

Comment:
This comment is a bit confusing, since it says "if ... didn't throw", but this also prevents overwriting the exception if it did throw. Might be better to simply say
```java
// ignore errors while draining and closing the stream